src/corelibs/U2View/src/ov_msa/MSAEditor.h

```diff
@@ -136,6 +136,20 @@ class U2VIEW_EXPORT MSAEditor : public MaEditor {
      */
     static constexpr int MAX_SUPPORTED_MSA_OBJECT_LENGTH = 100 * 1000 * 1000;
 
+    MaEditorMultilineWgt *getUI() const override {
+        return getMaEditorMultilineWgt();
+    }
+
+    MaEditorWgt *getMaEditorWgt(uint index = 0) const override {
+        return qobject_cast<MsaEditorWgt *>(getUI()->getUI(index));
+    }
+
+    MaEditorMultilineWgt *getMaEditorMultilineWgt() const override {
+        return qobject_cast<MsaEditorMultilineWgt *>(ui);
```

Comment:
Can this method return nullptr? if not, add a safepoint + document this feature. If yes - check all caller sites that nullptr is handled correctly.

Reply:
It can
MaEditor class tried to use it in font metric calculation